### ISSUE TITLE
User reports being unable to effectively use extension with screen reader

### ISSUE THREAD

**Issue by jeffbl** [Shared-Reality-Lab/IMAGE-browser#259]
User who we meet with Wednesday mornings reports that using mouse, can use extension fine, but when tries to do it completely unsighted with keyboard, is unable to activate IMAGE. Not clear if this is a keyboard shortcut issue for bringing up context menu, or something else. Need to track down actual issue and determine if it is technical, more explanation is needed, or there is a UI change that needs to be made so that it is clear to user how to trigger IMAGE, and they are able to do so.

**Comment by jeffbl** [comment]
This is the earlier issue, but closing as duplicate of later issue logged by @Sabrina-Knappe, which has more details.
